framework/src/araui/libs/vec.c

```diff
@@ -0,0 +1,113 @@
+/** 
+ * Copyright (c) 2014 rxi
+ *
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT license. See LICENSE for details.
+ */
+
+#include "vec.h"
+
+
+int vec_expand_(char **data, int *length, int *capacity, int memsz) {
```

Comment:
In this file, indentation seems to be somewhat different from others.
As this file is under /framework, it should follow the coding rule.

Reply:
Fixed